src/stores/CatalogOnlineQuery/CatalogOnlineQueryConfigStore.ts

```diff
@@ -54,6 +54,8 @@ export class CatalogOnlineQueryConfigStore {
         this.vizierResource = new Map();
         this.vizierKeyWords = "";
 
+        this.resetSearchRadius();
+
```

Comment:
Once `this.resetSearchRadius()` is setted, does below reaction section become redundent?

Reply:
The reaction is required for reseting the inputs again when the active frame is changed.